ftp/src/test/java/akka/stream/alpakka/ftp/FtpBaseSupport.java

```diff
@@ -49,6 +49,22 @@
         }
     }
 
+    public void cleanFiles() {
+        for (Path rootDir : getFileSystem().getRootDirectories()) {
```

Comment:
LOL. Of course, it's a virtual file system based on [jimfs](https://github.com/google/jimfs).